### ISSUE TITLE
Stats not updating on annotation delete

### ISSUE THREAD

**Issue by mattmcgrattan** [dlcs/elucidate-server#68]
It looks like the stats are not being updated when annotations are deleted.

Stats request:

[https://nlw-elucidate.digtest.co.uk/annotation/w3c/services/stats/target?field=id
](https://nlw-elucidate.digtest.co.uk/annotation/w3c/services/stats/target?field=id)

```json
{
"value": "http://dams.llgc.org.uk/iiif/2.0/4064120/canvas/4064123.json",
"total": 79
}
```

But:

[https://nlw-elucidate.digtest.co.uk/annotation/w3c/services/search/target?fields=source,id&value=http://dams.llgc.org.uk/iiif/2.0/4064120/canvas/4064123.json](https://nlw-elucidate.digtest.co.uk/annotation/w3c/services/search/target?fields=source,id&value=http://dams.llgc.org.uk/iiif/2.0/4064120/canvas/4064123.json)

```json
{
"@context": [
"http://www.w3.org/ns/anno.jsonld",
"http://www.w3.org/ns/ldp.jsonld"
],
"id": "https://nlw-elucidate.digtest.co.uk/annotation/w3c/services/search/target?fields=source%2Cid&value=http%3A%2F%2Fdams.llgc.org.uk%2Fiiif%2F2.0%2F4064120%2Fcanvas%2F4064123.json",
"type": "AnnotationCollection",
"first": {
"type": "AnnotationPage",
"as:items": {
"@list": []
},
"partOf": "https://nlw-elucidate.digtest.co.uk/annotation/w3c/services/search/target?fields=source%2Cid&value=http%3A%2F%2Fdams.llgc.org.uk%2Fiiif%2F2.0%2F4064120%2Fcanvas%2F4064123.json",
"startIndex": 0
},
"last": "https://nlw-elucidate.digtest.co.uk/annotation/w3c/services/search/target?page=0&fields=source%2Cid&value=http%3A%2F%2Fdams.llgc.org.uk%2Fiiif%2F2.0%2F4064120%2Fcanvas%2F4064123.json&desc=1",
"total": 0
}
```

**Comment by danielgrant** [comment]
Could be the same underlying issue as #61.

**Comment by danielgrant** [comment]
This was resolved in #53, release 1.4.1.
